common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/ShapleyLoadOption.java

```diff
@@ -0,0 +1,41 @@
+package ai.h2o.mojos.deploy.common.transform;
+
+/**
+ * Enum defining options for loading the mojo to enable Shapley predictions.
+ */
+public enum ShapleyLoadOption {
+  ALL,
+  NONE,
+  ORIGINAL,
+  TRANSFORMED;
+
+
+  /**
+   * Checks whether Shapley scoring is permitted.
+   * @param option {@link ShapleyLoadOption}
+   * @return {@link Boolean}
+   */
+  public static boolean isEnabled(ShapleyLoadOption option) {
+    switch (option) {
+      case ALL:
+      case ORIGINAL:
+      case TRANSFORMED:
+        return true;
+      case NONE:
+      default:
+        return false;
+    }
+  }
+
+  /**
+   * Checks whether requested type of Shapley value scoring is permitted.
+   * @param requested {@link String}
+   * @return {@link Boolean}
+   */
+  public static boolean requestedTypeEnabled(ShapleyLoadOption option, String requested) {
+    if (option.equals(ALL)) {
+      return true;
+    }
+    return option.name().equals(requested);
```

Comment:
Not required, just adding because it's sometimes fun to see how concise a return statement can be:

`return option == ALL || option.name().equals(requested);`

But arguably easier to read as you have it.  Just sharing.

Reply:
hehe so when I have to parse `||` in my head it immediately becomes harder to read :rofl:

Reply:
so yea I'm going to stick with readability.